app/controllers/FeedbackFormsResponseController.scala

```diff
@@ -188,6 +195,8 @@ class FeedbackFormsResponseController @Inject()(messagesApi: MessagesApi,
             feedbackResponseRepository.upsert(feedbackResponseData).map { result =>
               if (result.ok) {
                 Logger.info(s"Feedback form response successfully stored for session ${feedbackFormResponse.sessionId} for user ${request.user.email}")
+                emailManager ! EmailActor.SendEmail(
+                  List(request.user.email), fromEmail, "Feedback Response", views.html.emails.feedbackresponse(header.email, header.topic, header.meetUp).toString)
```

Comment:
Please change subject "Feedback Response" to "Feedback Successfully Registered!"